src/models/multiBarChart.js

```diff
@@ -179,11 +179,6 @@ nv.models.multiBarChart = function() {
             if (showLegend) {
                 legend.width(availableWidth - controlWidth());
 
-                if (multibar.barColor())
-                    data.forEach(function(series,i) {
-                        series.color = d3.rgb('#ccc').darker(i * 1.5).toString();
-                    });
```

Comment:
ah, I see now.  That seems reasonable.  Okay, I'll give it a look over as soon as I can, about to have to run out for a while though :)